sass/patternfly-react.scss

```diff
@@ -21,3 +21,61 @@ $icon-font-path: '~patternfly/dist/fonts/';
 */
 
 @import 'patternfly-react/patternfly-react';
+
+/** Following is used by SerialConsole */
```

Comment:
This should go in `sass/patternfly-react/_serialConsole.sass` and then add `@import 'serialConsole'` to `sass/patternfly-react/_patternfly-react.scss`

Reply:
done